client/my-sites/email/email-providers-stacked-comparison/provider-cards/utils.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+	RequestCartProduct,
+	ResponseCart,
+	ShoppingCartManagerActions,
+} from '@automattic/shopping-cart';
+import page from 'page';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+import { IncompleteRequestCartProduct } from 'calypso/lib/cart-values/cart-items';
+
+export enum IntervalLength {
+	ANNUALLY = 'annually',
+	MONTHLY = 'monthly',
+}
+
+export const addToCartAndCheckout = (
+	shoppingCartManager: ShoppingCartManagerActions,
+	cartItem: RequestCartProduct | IncompleteRequestCartProduct,
+	productList: Record< string, { product_id: number } >,
+	setAddingToCart: ( addingToCart: boolean ) => void,
+	selectedSite: string
+): void => {
```

Comment:
Nit: this will be simpler to maintain if we supply an object with named properties.
```suggestion
export const addToCartAndCheckout = ( {
	shoppingCartManager: ShoppingCartManagerActions,
	cartItem: RequestCartProduct | IncompleteRequestCartProduct,
	productList: Record< string, { product_id: number } >,
	setAddingToCart: ( addingToCart: boolean ) => void,
	selectedSite: string
} ): void => {
```